examples/load_certs.py

```diff
@@ -52,29 +60,29 @@ def cert_id_exists_in_database(log_id):
     return False
 
 
-def get_new_log_ids(domain, max_expired_date, verbose=False):
-    """Generate a sequence of new CT Log IDs.
+def get_new_log_issuances(domain, max_expired_date, verbose=False):
+    """Generate a sequence of new CT Log issuances.
 
     Arguments:
     domain -- the domain name to query
     max_expired_date -- a date to filter out expired certificates
 
-    Yields a sequence of new, unique, log IDs.
+    Yields a sequence of new, unique, log issuances.
 
     """
     if verbose:
         tqdm.write(f"requesting certificate list for: {domain}")
-    expired = domain != "nasa.gov"  # NASA is breaking the CT Log
-    cert_list = summary_by_domain.delay(domain, subdomains=True, expired=expired)
+
+    cert_list = summary_by_domain.delay(domain, subdomains=True)
     cert_list = cert_list.get()
     duplicate_log_ids = set()
     for i in tqdm(cert_list, desc="Subjects", unit="entries", leave=False):
         log_id = i["id"]
+        name = i["dns_names"][0]  # take the first dns name available
```

Comment:
We strive for proper capitalization whenever possible:
```suggestion
        name = i["dns_names"][0]  # take the first DNS name available
```